_posts/0000-01-02-TobiTobTobsen.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
use a picture or a link to make it dynamik, here is an example
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)